dasharo-onboarding.md

```diff
@@ -254,26 +278,105 @@ Sonoff IP:
 PiKVM IP:
 PiKVM HW Base:
 ```
+
 Checkout the platform:
+
 ```
 ./osfv_cli.py snipeit check_out --rte_ip 192.168.10.233
 ```
+
+---
+
+### Connecting and basic operations - cont.
+
 Now if you run `./osfv_cli.py snipeit list_used | grep VP2410 -A 8` you should get the same entry of Protectli as above.
 
 You can get list of possible operations by running
+
 ```
 ./osfv_cli.py rte --rte_ip 192.168.10.233 -h
 ```
+
 For example you can check the state of the GPIO pin 0:
+
 ```
 ./osfv_cli.py rte --rte_ip 192.168.10.233 gpio get 0
 ```
 
-## Exercises
+Access the platform by serial interface:
 
+```
+./osfv_cli.py rte --rte_ip 192.168.10.233 serial
+```
 
+Fetch current ROM image:
+
+```
+./osfv_cli.py rte --rte_ip 192.168.10.233 flash read --rom vp2410-read.rom
+```
 
 ---
-class: center, middle, outro
 
+### Connecting and basic operations - cont.
+
+Writing new ROM image:
+
+```
+./osfv_cli.py rte --rte_ip 192.168.10.233 flash write --rom ~/coreboot/protectli_vault_glk_v1.0.15.rom
+```
+
+Switching power on relay:
+```
+./osfv_cli.py rte --rte_ip 192.168.10.233 rel tgl
+```
+
+---
+
+# Building firmware
+
+This can be done as shown in [building manual](https://docs.dasharo.com/variants/protectli_vp2410/building-manual/).
+
+Tips:
+- In protectli blobs repository make sure you are on the branch with the same name as in coreboot repository
+- Instead of creating symbolic link you can simply copy Geminilake (inside root coreboot directory):
+
+    ```
+    cp -r 3rdparty/blobs/mainboard/protectli/vault_glk/GeminilakeFspBinPkg 3rdparty/fsp/GeminilakeFspBinPkg
```

Comment:
Why do we suggest deviation from the documentation? Was there any problem with that?

Reply:
I had a problem with symlink - after creating it, build script wasn't working, I came up with an idea to copy it